test/NuGet.Client.Test.Apex/NuGetClientTests.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
Why do you have this solution?

Reply:
Based on the previous comments in https://github.com/NuGet/NuGet.Client/pull/2000.
Rohit said "Another thing to note is that we do not want Apex projects in NuGet.sln. If you need to put all the projects in one sln, you should create a new sln under the test directory."
So i create a new sln under it

Reply:
@rohit21agrawal's comment was related to the comment I left in the `sln` file in this PR. If this solution is not needed consider deleting it. What I usually do when working with apex is get the apex projects into the solution but never adding those changes to my commits.